testsuite/tests/prometheus/backend_listener/test_backend_listener_api.py

```diff
@@ -135,12 +136,12 @@ def _auth_request(method, endpoint, auth_object):
     return _auth_request
 
 
-def authrep_backend_api_query(request_type):
+def authrep_backend_api_query(request_type) -> Tuple[str, Dict[str, str]]:
     """
-    Returns prometheus backend listener enpoint codes
-    query for given request type
+    Returns prometheus query parts for the apisonator_listener_response_codes metric
+    with the request type passed in the parameters
     """
-    return f"apisonator_listener_response_codes{{request_type=\"{request_type}\"}}"
+    return "apisonator_listener_response_codes", {"request_type": request_type}
```

Comment:
even new interface would remain compatible with this, wouldn't be?

Reply:
not really. When user-workload is used, multiple 3scale instances may be gathered so new implementation adds namespace label into query.